_layouts/bio.html

```diff
@@ -19,6 +19,7 @@ <h1 class="font-heading-3xl text-base-lightest">{{ page.title }}</h1>
                 </div>
                 <div class="tablet:grid-col-7">
                     {{ content }}
+                    <button class="usa-button">Connect on LinkedIn</button>
```

Comment:
I still think button is the right approach, but I didn't think it all the way through... Now we have some content in the layout. We have at least a couple options: a) use front matter for the button label, or b) move the HTML button markup into the markdown file. Both have their plusses and minuses. What do you think?

Reply:
Front matter would probably be best for our team's "reusability" standards 😂  I'll make changes to this 👍🏻